src/main/webapp/calendar-script.js

```diff
@@ -22,45 +22,25 @@ $(document).ready(() => {
       right: 'dayGridMonth, timeGridWeek, timeGridDay, listWeek'
     },
     initialView: 'timeGridWeek',
-    initialDate: '2020-06-29',
-    eventLimit: true,
-    events: [{
-            title: 'Golden Gate Bridge',
-            start: '2020-06-29T09:00:00',
-            end: '2020-06-29T10:00:00',
-            allDay: false
-        },
-        {
-            title: 'Fishermans Wharf',
-            start: '2020-06-29T10:30:00',
-            end: '2020-06-29T11:30:00',
-            allDay: false
-        },
-        {
-            title: 'Mission District',
-            start: '2020-06-29T12:00:00',
-            end: '2020-06-29T13:00:00',
-            allDay: false
-        },
-        {
-            title: 'Lunch Time',
-            start: '2020-06-29T13:00:00',
-            end: '2020-06-29T14:00:00',
-            allDay: false
-        },
-        {
-            title: 'Sutro Tower',
-            start: '2020-06-29T14:30:00',
-            end: '2020-06-29T15:30:00',
-            allDay: false
-        },
-        {
-            title: 'West Portal Neighborhood',
-            start: '2020-06-29T16:00:00',
-            end: '2020-06-29T17:00:00',
-            allDay: false
-        },
-    ]
+    navLinks: true,
+    dayMaxEvents: true, //alow "more" link when too many events on one day
   });
+  getEvents(calendar);
   calendar.render();
 });
+
+/**
+ * retrives the events from /calculate-trip url and dynamically adds the events
+ */
+function getEvents(calendar) {
+  fetch('/calculate-trip').then(response => response.json()).then((events) => {
```

Comment:
This is fine right now, just adding because I have a question about future direction: should we consolidate the GET function for Maps, Calendar, and the future Edit page under `/calculate-trip`, or should we have separate servlets for getting the relevant information and formatting it for all of these pages (i.e.; maps fetches `/get-maps`, calendar fetches `/get-calendar`, etc)?

Reply:
I think probably separate servlets!

Reply:
I've done this in my later CL where I also Post and Get TripDays